src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/StringExtensions.cs

```diff
@@ -1,4 +1,6 @@
 #nullable enable
+using AngleSharp.Text;
+using NaturalSort.Extension;
```

Comment:
Is `NaturalSort.Extension` being used?  Couldn't immediately see its use below, but such is the nature of extension methods!